src/core/client.ts

```diff
@@ -72,10 +84,15 @@ export class OakConsentClient {
     if (typeof window !== "undefined") {
       window.oakConsent = this;
     }
-    this.isReady = this.init();
```

Comment:
This was being triggered on the server side during SSR in OWA which isn't ideal. I've left it up to the consumer to call it once (which i'm doing in `ConsentProvider` in a `useEffect` so it only gets initialised client-side)

Reply:
Good idea